deps/ncrypto/ncrypto.cc

```diff
@@ -220,4 +221,75 @@ int PasswordCallback(char* buf, int size, int rwflag, void* u) {
   return -1;
 }
 
+// ============================================================================
+// SPKAC
+
+bool VerifySpkac(const char* input, size_t length) {
+#ifdef OPENSSL_IS_BORINGSSL
+  // OpenSSL uses EVP_DecodeBlock, which explicitly removes trailing characters,
+  // while BoringSSL uses EVP_DecodedLength and EVP_DecodeBase64, which do not.
+  // As such, we trim those characters here for compatibility.
+  length = std::string(input.data()).find_last_not_of(" \n\r\t") + 1;
```

Comment:
find last not of can return npos which makes this UB. can we add a check here somewhere?

Reply:
That's why the `+ 1` is there I presume. This should cause the returned `npos` to roll over to 0 should it not?